tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- include: packages.yml
+  when: (not openssl_certs_stat.results[0].stat.exists or
+        not openssl_certs_stat.results[1].stat.exists) and
+        not openssl_node_only_ca
```

Comment:
I would very much like to remove this conditional logic. @msheiny can you explain the utility here? I think this include should _always_ run, and it'll do so idempotently automatically.